website/dbt-global-variables.js

```diff
@@ -16,5 +16,17 @@ exports.dbtVariables = {
         "version": "0.21"
       }
     ]
-  } 
+  },
+    dbt: {
+    name: "dbt"
+  },
+    dbtCore: {
+    name: "dbt Core"
+  },
+    dbtCloud: {
+    name: "dbt Cloud"
+  },
+    dbtIDE: {
+    name: "dbt IDE"
```

Comment:
Is this the official name? Wouldn't it be the dbt Cloud IDE? 

But I guess the magic of variables is that we can change our mind on this later